src/lib/updates/profileFix.js

```diff
@@ -31,7 +31,7 @@ const checkProfile = async (lastUpdate, prevVersion, log) => {
         Promise.race([
           // set retry & timeout for each field separately
           flushField(field),
-          timeout(1000, `fixProfile: '${field}' field flush timeout`),
+          timeout(2500, `fixProfile: '${field}' field flush timeout`),
```

Comment:
on Nokia 2.4 (an i suppose on the other low-end smartphones) Gun runs slowly. 1sec isn't enough for .decrypt operation. So i increased timeout because i always got fixProfile exception